sdk/core/azure-core/CMakeLists.txt

```diff
@@ -8,11 +8,14 @@ project(${TARGET_NAME} LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-set(CURL_MIN_REQUIRED_VERSION 7.4)
+# min version for `CURLSSLOPT_NO_REVOKE`
+# https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
+set(CURL_MIN_REQUIRED_VERSION 7.44) 
```

Comment:
Need 7.44 for No Revoke option for libcurl.

I don't remember why we choose 7.4  It might be the default version in some Linux distribution of something.